### PR TITLE
Timeout in seconds, not milliseconds.

### PR DESCRIPTION
**Pull request by breerly**
- update docs to show timeouts as seconds
- create integration test verifying timeout
- update default timeout to be 30 seconds, not 16 minutes

### CODE REVIEW

**Comment by abhinav** [comment]
:+1:

**Comment by kriskowal** [comment]
Does this preclude ms resolution timeouts?

**Comment by junchaowu** [comment]
change the default value

**Comment by abhinav** [comment]
@junchaowu We don't want to change behavior in 0.17 yet. This just fixes the docs to reflect what the current behavior is.

**Comment by abhinav** [comment]
:+1:

File: tchannel/sync/client.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ class TChannel(AsyncTChannel):
         # thrift_service is the result of a call to ``thrift_request_builder``
         future = tchannel.thrift(
             thrift_service.getItem('foo'),
-            timeout=1000,  #  1 second
+            timeout=2,  #  2 seconds
```
Comment (blampe) [inline tchannel/sync/client.py]: 0.5

**Comment by blampe** [comment]
Thanks for jumping on this :+1:

File: tchannel/sync/client.py (hunk 1 above)
Comment (breerly) [inline tchannel/sync/client.py]: thx

**Pull request merged**